WowPacketParserModule.V9_0_1_36216/Parsers/ScenarioHandler.cs

```diff
@@ -0,0 +1,39 @@
+﻿using WowPacketParser.Enums;
+using WowPacketParser.Misc;
+using WowPacketParser.Parsing;
+
+namespace WowPacketParserModule.V9_0_1_36216.Parsers
+{
+    public static class ScenarioHandler
+    {
+        [Parser(Opcode.SMSG_SCENARIO_STATE)]
+        public static void HandleScenarioState(Packet packet)
+        {
+            packet.ReadInt32("ScenarioID");
+            packet.ReadInt32("CurrentStep");
+            packet.ReadInt32<DifficultyId>("DifficultyID");
```

Comment:
All fields between DifficultyID - SpellsCount (also including both) should be uint32

Reply:
Was copied from struct in WPP (will change)